parsimonious/grammar.py

```diff
@@ -95,8 +87,9 @@ def _copy(self):
 
         """
         new = Grammar.__new__(Grammar)
+        super(Grammar, new).__init__(self.items())
```

Comment:
Can we just do a simple `new = Grammar(self.items())` rather than reaching all the way down to `__new__`?